README.md

```diff
@@ -22,6 +28,20 @@ The intention of Safe Harbor language provided by disclose.io is for it to be fo
 
 Policies missing any of the core tenets above, but that still contain a good-faith statement committing not to pursue legal action on security researchers, meet the criteria for **Partial Safe Harbor**.
 
+### Disclosure types
+
```

Comment:
Is it worth calling out here that these are not an either/or with VDP/BBPs? But rather, you can have a BBP or VDP together with one of these disclosure types?

Reply:
Interesting idea. How would you articulate that distinction more clearly in the language? Remember, one of the goals is to minimize words where possible (but I also agree with your statement).

Reply:
@charlieeriksen A thought here would be that public open-scope bug bounty programs are unlikely to coexist with an open-scope VDP - Are you referring to closed-scope or private programs here?

Reply:
Policies missing any of the core tenets above, but that still contain a good-faith statement committing not to pursue legal action on security researchers, _may_ meet the criteria for Partial Safe Harbor. _Incentives or "bounties" for vulnerability disclosures are not a prerequisite._